doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc

```diff
@@ -2,6 +2,10 @@
 
 = Heterogeneous Node Classification with HashGNN and Autotuning
 
+This Jupyter notebook is hosted
+https://github.com/neo4j/graph-data-science-client/blob/main/examples/heterogeneous-node-classification-with-hashgnn.ipynb[here]
+in the Neo4j Graph Data Science Client Github repository.
+
```

Comment:
These appear now above the `open in colab` button. I prefer having the button on top.

Reply:
![Screenshot 2023-02-28 at 10 52 23](https://user-images.githubusercontent.com/2419675/221816926-8a1b7cfd-da3d-4813-9052-2fb225f59167.png)

Reply:
Can we move it down one cell perhaps? And update instructions?

Reply:
Ok :) I prefer it slightly the other way (why I made it so) but don't have a strong opinion. So I'll move it then

Reply:
Done!